docs/vyos.vyos.vyos_ntp_global_module.rst

```diff
@@ -210,9 +211,9 @@ Examples
     # # Before state:
     # # -------------
     #   vyos@vyos:~$ show configuration commands | grep ntp
-    #     set system ntp server time1.vyos.net
-    #     set system ntp server time2.vyos.net
-    #     set system ntp server time3.vyos.net
+    #     set system/service ntp server time1.vyos.net
```

Comment:
For these, please just use the newer `service`. This is intended to be a reproducible example and it makes sense that we have these using the most recent code instead of `system/service` which won't be representative of either output directly./